addons/l10n_ca/data/fiscal_templates_data.xml

```diff
@@ -932,28 +922,28 @@
     <record id="fiscal_position_tax_template_pe2bc_sale_en" model="account.fiscal.position.tax.template">
         <field name="position_id" ref="fiscal_position_template_bc_en"/>
         <field name="tax_src_id" ref="hst15_sale_en"/>
-        <field name="tax_dest_id" ref="gst_sale_en"/>
+        <field name="tax_dest_id" ref="gstpst_bc_sale_en"/>
     </record>
 
     Already created by ns2mb_sale
     <record id="fiscal_position_tax_template_pe2mb_sale_en" model="account.fiscal.position.tax.template">
         <field name="position_id" ref="fiscal_position_template_mb_en"/>
         <field name="tax_src_id" ref="hst15_sale_en"/>
-        <field name="tax_dest_id" ref="gst_sale_en"/>
+        <field name="tax_dest_id" ref="gstpst_mb_sale_en"/>
     </record>
 
     Already created by ns2nb_sale
     <record id="fiscal_position_tax_template_pe2nb_sale_en" model="account.fiscal.position.tax.template">
         <field name="position_id" ref="fiscal_position_template_nb_en"/>
         <field name="tax_src_id" ref="hst15_sale_en"/>
-        <field name="tax_dest_id" ref="hst13_sale_en"/>
+        <field name="tax_dest_id" ref="hst15_sale_en"/>
     </record>
```

Comment:
You get the idea, no need to keep those comments at all if they map A to A

Reply:
I do, still, useless to change anything related as the file will be dropped later